server/aesdsocket.c

```diff
@@ -14,140 +14,288 @@
 #include<string.h>
 #include<arpa/inet.h>
 #include<netinet/in.h>
+#include "queue.h"
```

Comment:
@Sona-Subair you shouldn't check the binary file `server/aesdsocket` into your repo.